pkgs/by-name/fl/flawz/package.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, openssl
+, sqlite
+, installShellFiles
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "flawz";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "orhun";
+    repo = "flawz";
+    rev = "v${version}";
+    hash = "sha256-cMieoxyTk+B4UkT/GFemnouvtVlhukmtu5e9S7KjM0k=";
+  };
+
+  cargoHash = "sha256-oz8jA2p2dHHryGbgAEtBPSxQSKQiLHbpfMFdo1LjWos=";
+
+  nativeBuildInputs = [ pkg-config installShellFiles ];
+
+  buildInputs = [ openssl sqlite ];
```

Comment:
Try adding `lib.optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks.SystemConfiguration ]`

Reply:
am a poor guy, i can't test this, but seems reasonable tho.

i see that there a CI's jobs related to `darwin` but it takes a long times to run, and i sorta wanna get this pr merged so i can start package another packages

Reply:
Ah, looks like there's also `darwin.apple_sdk.frameworks.CoreServices`... If you don't want to add support for darwin this time, just add `broken = stdenv.isDarwin; # needing some apple_sdk packages` to `meta`.

Reply:
>  just add broken = stdenv.isDarwin; # needing some apple_sdk packages to meta.

i dont have any way to test and see if it'll build on macOS or not, so I guess this's the only option for now (unless someone who have mac appears and offers help)